Use Australia National Greenhouse Accounts inventory

aethr opened the pull request:
## Description

Resolves #73.

Replaces the undocumented `ch4-sectoral-emissions.csv` input file with data from Australian National Greenhouse Accounts "Paris Agreement" inventory, broken down by UNFCCC sector.

The work consistutes two major parts:
- map from inventory UNFCCC category names to category codes
- search through the inventory for emissions matching a list of sectors to compute a sector total

## Checklist

Please confirm that this pull request has done the following:

- [ ] Tests added
- [ ] Documentation added (where applicable)
- [ ] Changelog item added to `changelog/`

## Notes

aethr commented:
I've constructed a test that is meant to check new inventory totals (using 2023 as a reference year) against the previous totals from `ch4-sectoral-emissions.csv`. Most of the sectors look very close, but `agriculture` and `fugitive` are alarmingly different.

```
fugitive: 0.0429 from 1.1739
agriculture: 0.2187 from 2.2825
electricity: 0.0111 from 0.0143
stationary: 0.0466 from 0.0529
waste: 0.4783 from 0.4729
transport: 0.0115 from 0.0114
industrial: 0.0029 from 0.0029
lulucf: 0.4869 from 0.51
```

prayner commented:
Agreed, the only one which surprises me is lulucf. fwiw the most
useful form for these numbers for me is to multiply that mean by the
number of grid cells and the cell_area then divide by 1e3 to get
tonnes of methane. Also useful to take the absolute difference which
is the number I focus on most. But, in general, looking good I think.

aethr commented:
> sector/inventory.py:47 seems some confusion between kilotonne and gigatonne

Fixed in [81b26ce](https://github.com/openmethane/openmethane-prior/pull/122/commits/81b26ce60812d397d9d032f7fbfec143708a7d33).

> sector/inventory.py:69 float(str) seems to work even on scientific notation

Fixed in [46e8673](https://github.com/openmethane/openmethane-prior/pull/122/commits/46e8673cffdcab8a327e843e8e3bb5ed2bec0913).
